eng/ReferenceAssemblies.targets

```diff
@@ -0,0 +1,16 @@
+<Project>
+  <PropertyGroup >
+    <TargetRefPath>$([MSBuild]::NormalizePath('$(ReferenceAssemblyOutputPath)', '$(MSBuildProjectName)', '$(Configuration)', '$(TargetName)$(TargetExt)'))</TargetRefPath>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(ProduceReferenceAssembly)' == 'true'">
```

Comment:
No I'm not using this but it took me a while to figure the right setup out. I kept it in case we ever want to enable reference assembly generation by the compiler. I can extract it out of this PR and submit another one with this change. Thoughts?

Reply:
We don't commit code that isn't in use: it'll just get stale.  You can put it in a branch somewhere and reference it from an issue.  If you have a scenario for using this feature now we can talk about it.  I've tried using it in a few places with mixed-results.

Reply:
sounds good to me. In case I ever need it again I can just look back at this PR.